### ISSUE TITLE
Leftover bad argument?

### ISSUE THREAD

**Issue by bilderbuchi** [DRMacIver/shrinkray#13]
If you hit the code path 
https://github.com/DRMacIver/shrinkray/blob/85a08a67fa2302588ed8372f643629ab7f83ac23/src/shrinkray/__main__.py#L474

you get "TypeError: ShrinkRayState.run_script_on_file() got an unexpected keyword argument 'test_case'". Maybe this is left over from some refactoring? I think that argument can just be deleted, but I am very much not sure.

**Comment by bilderbuchi** [comment]
Also, arguably, in this section
https://github.com/DRMacIver/shrinkray/blob/85a08a67fa2302588ed8372f643629ab7f83ac23/src/shrinkray/__main__.py#L452-L455

`run_for_exit_code` should probably get `debug=True`, because otherwise this does not serve _debugging purposes_ well, I think.

**Comment by DRMacIver** [comment]
Yeah, these both look right. Sorry, I should have way more testing for all those code paths than I currently do.

**Comment by bilderbuchi** [comment]
Ah, no worries! ^^

**Comment by DRMacIver** [comment]
Thanks for reporting these. All fixed now.
